### PR TITLE
Making Animal Telemetry in schedule actionable

### PR DESCRIPTION
**Pull request by timvdstap**
Proposed change to make `Animal Telemetry` in the schedule actionable.

### CODE REVIEW

File: _includes/custom-schedule.html [hunk 1]
```diff
@@ -65,7 +65,7 @@ <h3>April 23<sup>rd</sup></h3>
             <ul>Creating the Schema</ul>
             <ul>QA/QC, Metadata and Publishing</ul>
             <ul>DNA Derived Data</ul>
-            <ul><a hrf="https://ioos.github.io/bio_mobilization_workshop/special-data-type-acoustic-telemetry/index.html">Animal Telemetry</a></ul>
+            <ul><td><a hrf="{{ site.url }}{{ site.baseurl }}/special-data-type-acoustics-telemetry/index.html">Animal Telemetry</a></td></ul>
```
Comment (MathewBiddle) [inline _includes/custom-schedule.html:68]: this should be `/special-data-type-acoustic-telemetry/` to reference https://ioos.github.io/bio_mobilization_workshop/special-data-type-acoustic-telemetry/index.html

File: _includes/custom-schedule.html [hunk 2]
```diff
@@ -91,7 +91,7 @@ <h3>April 23<sup>rd</sup></h3>
...
```
Comment (MathewBiddle) [inline _includes/custom-schedule.html:94]: this should be `/special-data-type-acoustic-telemetry/` to reference https://ioos.github.io/bio_mobilization_workshop/special-data-type-acoustic-telemetry/index.html

**Review by MathewBiddle — CHANGES_REQUESTED** [review]
This url will not resolve with this change as it stands.

File: _includes/custom-schedule.html (hunk 1 above)
Comment (timvdstap) [reply _includes/custom-schedule.html]: Changed acoustics to acoustic.

**Comment by timvdstap** [comment]
> This url will not resolve with this change as it stands.

I think should be resolved now? I changed acoustic**s** to acoustic in the URL.

**Review by MathewBiddle — APPROVED** [review]

**Pull request merged**